### PR TITLE
feat: feedback form

### PR DESCRIPTION
**Pull request by alexiscolin**
related to https://github.com/gnolang/gno/issues/1505, this PR aims to add a feedback form to the Gno Doc

### CODE REVIEW

**Comment by leohhhn** [comment]
@alexiscolin thanks for this.
I have a couple of points in mind regarding this:
- Where do we want to redirect the user when they submit feedback? Maybe, we should make the feedback form into a "Thanks for feedback" (basically not redirect them anywhere)
- It would be great if its possible to include in the feedback data, the index/ID/URL, maybe even commit, of the page they left the feedback on. This way we know exactly what version of which page they left the feedback on
- I will ask Ops where to store the feedback data.
-

**Comment by alexiscolin** [comment]
@leohhhn I've linked the form to the form ;)

**Review by leohhhn — APPROVED** [review]
Nice! 

Would be great to get some more validation in the fields - email and feedback fields cannot not be empty when the user sends feedback.

Also, I see that the Page ID in the Google Sheet is not relayed yet, it only says "/" for every feedback that is sent. 

About the page commit, it's a nice to have, so lets see if it's too complicated to implement or not.

**Comment by alexiscolin** [comment]
@leohhhn I've added the page commit from Github API and set the fields as `required` to ensure all of them are filled before shipping the content. Regarding the page url, that's weird, if you look at the result in gform, you will see that it looks ok with mine. Is it still the case for you?

File: docusaurus/src/components/Feedback/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  };
+
+  const [formData, setFormData] = useState({
+    email: "",
+    feedback: "",
+  });
+
+  const feedbackMsg = {
+    info: "Leave feedback",
+    callback: "Thank you for your feedback",
```
Comment (leohhhn) [inline docusaurus/src/components/Feedback/index.js:18]:
```suggestion
    callback: "Thank you for your feedback! ❤️",
```

File: docusaurus/src/components/Feedback/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      <div className="footer__title">Was this page helpful?</div>
+      {isOpen ? (
+        <form className="feedback__form" onSubmit={sendData}>
+          <button className="feedback__inner-btn feedback__close" aria-label="Close" onClick={toggle}>
+            ✕
+          </button>
+          <div>
+            <div className="feedback__field">
+              <label htmlFor="feedback-email">Email</label>
+              <input type="email" name="email" id="feedback-email" placeholder="albert@email.com" value={formData.email} onChange={handleInputChange} required />
```
Comment (leohhhn) [inline docusaurus/src/components/Feedback/index.js:95]:
```suggestion
              <input type="email" name="email" id="feedback-email" placeholder="example@domain.com" value={formData.email} onChange={handleInputChange} required />
```

**Review by leohhhn — APPROVED** [review]
💯

**Pull request merged**